src/databricks/azext_databricks/commands.py

```diff
@@ -13,18 +13,27 @@
 
 def load_command_table(self, _):
 
-    from ._client_factory import cf_workspaces
-
-    with self.command_group('databricks', is_preview=True):
-        pass

```

Comment:
Just to confirm, preview tag removed to target GA?

Reply:
@fengzhou-msft I saw the message from Veera "Yes after adding support for vNet Peering both PS and Az CLI can be released by removing experimental tags. ". I will double-check this with him.

Reply:
Yes you can remove Preview and release. However keep the Preview for CMK related options in documentation and help messages as discussed